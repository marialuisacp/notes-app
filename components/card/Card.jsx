import React from 'react';
import StyledCard from './Card.styles';

const Card = ({ text }) => (
  <StyledCard> {text} </StyledCard>
);

export default Card;