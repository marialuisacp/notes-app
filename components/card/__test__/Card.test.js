import React from 'react';
import '@testing-library/jest-dom';
import { render, screen } from '@testing-library/react';
import Card from '../Card';

const defaultProps = {
  text: 'Teste de card',
  labels: ['red']
};

test('should render component Card', () => {
  const { CardFragment } = render(<Card text={defaultProps.text}></Card>);
  expect(CardFragment).toMatchSnapshot();
})

test('should have default text in component Card', () => {
  render(<Card text={defaultProps.text}></Card>);

  expect(screen.getByText(defaultProps.text)).toBeInTheDocument();
})