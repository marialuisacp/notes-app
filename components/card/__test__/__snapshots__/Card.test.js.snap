// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render component Card 1`] = `undefined`;
